src/main/java/com/cleanroommc/modularui/widget/Widget.java

```diff
@@ -420,4 +421,13 @@ public String toString() {
         }
         return getClass().getSimpleName();
     }
+
+    public String getInternalName() {
+        return internalName;
+    }
+
+    public W setInternalName(String internalName) {
```

Comment:
I still feel suspicious about this feature. I guess below should work as replacement for internal name thing
<img width="563" alt="internal" src="https://github.com/CleanroomMC/ModularUI/assets/40035906/21a69353-7152-44ce-aeb2-d5f0f456f91b">

Reply:
This is much smarter. will remove it